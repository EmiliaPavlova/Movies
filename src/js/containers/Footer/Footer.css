.footer {
  padding: 10px 80px;
  background: #414141;
}