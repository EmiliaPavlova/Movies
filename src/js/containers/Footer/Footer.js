import React, { Component } from 'react';
import Logo from '../LogoContainer/LogoContainer';
import './Footer.css';

class Footer extends Component {
  render() {
    return (
      <div className='footer'>
        <Logo />
      </div>
    )
  }
}

export default Footer;
