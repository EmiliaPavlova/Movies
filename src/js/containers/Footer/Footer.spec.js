import React from 'react';
import { mount } from 'enzyme';
import Footer from './Footer';
import './Footer.css';

const mountFooter = props => mount(<Footer {...props} />);

describe('Footer', () => {
  describe('Snapshot', () => {
    it('matches the snapshot', () => {
      const wrapper = mountFooter();
      expect(wrapper).toMatchSnapshot();
    })
  });
});
