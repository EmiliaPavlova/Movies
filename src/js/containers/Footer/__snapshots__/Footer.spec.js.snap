// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot matches the snapshot 1`] = `
<Footer>
  <div
    className="footer"
  >
    <LogoContainer>
      <Logo
        text="netflixroulette"
      >
        <div
          className="logo"
        >
          netflixroulette
        </div>
      </Logo>
    </LogoContainer>
  </div>
</Footer>
`;
